server/src/main/java/com/linecorp/centraldogma/server/internal/api/converter/WatchRequestConverter.java

```diff
@@ -81,6 +81,24 @@ public WatchRequest convertRequest(
         return new WatchRequest(lastKnownRevision, timeoutMillis, notifyEntryNotFound);
     }
 
+    String extractRevision(String ifNoneMatch) {
```

Comment:
```suggestion
    @VisibleForTesting
    String extractRevision(String ifNoneMatch) {
```

Reply:
I'm sorry but, is there a reason to annotate `@VisibleForTesting`?.?

Reply:
@TheWeaVer
To tell the reviewers that this method is only used by tests so we didn't make it private.  😄

Reply:
Ah.. I see. To announce that why this API can not be `private`. 🙂